deny.toml

```diff
@@ -0,0 +1,62 @@
+[advisories]
+db-path = "~/.cargo/advisory-db"
+db-urls = ["https://github.com/rustsec/advisory-db"]
+vulnerability = "deny"
+unmaintained = "deny"
+yanked = "deny"
+notice = "deny"
+ignore = [
+    "RUSTSEC-2020-0016", # net2 deprecated - https://github.com/deprecrated/net2-rs/commit/3350e3819adf151709047e93f25583a5df681091
+    "RUSTSEC-2020-0056", # stdweb unmaintained - https://github.com/koute/stdweb/issues/403
+    "RUSTSEC-2021-0047", # security issue - https://github.com/gnzlbg/slice_deque/issues/90
+]
+
+[licenses]
+unlicensed = "allow"
```

Comment:
https://github.com/vislyhq/stretch is MIT but not recognised by cargo deny or by crates.io and couldn't get an exception working for it

Reply:
glad you found a way around this. i feel much safer now :)

Reply:
I was following cargo deny doc that says you need to specify a file to hash-check it's the same version, but it couldn't find any file for this crate... after reading the source I realised the file to check can be... nothing and it will pass validation 😄